web/cobrands/makemyisland/layout.scss

```diff
@@ -34,10 +40,43 @@ body.frontpage #united-nations-dp-logo {
   z-index:3; // clear of the nav container
 }
 
+#front-main {
+    max-width: 55em; // 60em (.container width), minus 2x2em (padding), minus 2x0.5em (.content margin)
+    margin: 0 auto;
+    background-color: #ffffe1;
+    background-color: rgba(#ffffe1, 0.8);
+}
+
+#front-main,
+body.twothirdswidthpage .container .content,
+body.fullwidthpage .container .content {
+    border-radius: 0.3em;
+    padding: 2em;
```

Comment:
The padding causes the main content to overlap the sidebar on e.g. the Help page; I think reducing its width accordingly should be enough? Will try.